.eslintignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+examples
```

Comment:
can i ask what is the difference between ignoring it in `.eslintignore` and using `ignorePatterns`?

Reply:
There isn't a difference functionally according to their [docs](https://eslint.org/docs/latest/use/configure/ignore), so i'll remove this file. Thanks for pointing this out!

Reply:
Removed in [bf5470a](https://github.com/opengovsg/sgid-client/pull/50/commits/bf5470a124d74e0d4eca77d5b836be7c79b3b6c8)